packages/autocomplete/src/autocomplete.vue

```diff
@@ -196,6 +197,9 @@
         }
         this.debouncedGetData(value);
       },
+      handleChange(event) {
+        this.$emit('change', event.target.value);
```

Comment:
https://github.com/ElemeFE/element/issues/19243

Emitted event from el-input contains the value of the input, not the event. This code throws an error because target is undefined.

Reply:
fixed in #19200